mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -178,13 +226,22 @@ function BalanceItemRow ({
             displayType='text'
             renderText={(value) =>
               <>
-                <BalanceText
-                  dark={tailwind('text-gray-200')}
-                  light={tailwind('text-black')}
-                  style={tailwind('mr-2 flex-wrap')}
-                  testID={`${testID}_amount`}
-                  value={value}
-                />
+                <View style={tailwind('flex mr-2 leading-6')}>
+                  <BalanceText
+                    dark={tailwind('text-gray-200')}
+                    light={tailwind('text-black')}
+                    style={tailwind('flex-wrap')}
+                    testID={`${testID}_amount`}
+                    value={value}
+                  />
+                  {isBalancesDisplayed && (
```

Comment:
another thing to note here, what should we display if balances are very small? Ex. $0.0000006, currently we show $0.00. Might be misleading to users. Need to check with design team